sound/soc/sof/ipc4-topology.c

```diff
@@ -2398,6 +2398,8 @@ static int sof_ipc4_widget_setup(struct snd_sof_dev *sdev, struct snd_sof_widget
 	}
 
 	if (swidget->id != snd_soc_dapm_scheduler) {
+		int module_id = msg->primary & SOF_IPC4_MOD_ID_MASK;
```

Comment:
Can you be more specific with the commit title? Something like
`ASoC: SOF: ipc4-topology: Use different print for module/pipeline widget setup`